src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -4506,6 +4506,11 @@ void Unit::_ApplyAura(AuraApplication* aurApp, uint8 effMask)
 {
     Aura* aura = aurApp->GetBase();
 
```

Comment:
Can this be made more readable? It’s not very easy to read as it is. What does critter have to do with totem though?

Reply:
Also won’t this block totems from receiving *any* aura? What about stoneclaw totem?